packages/babel-plugin-proposal-private-methods/test/fixtures/private-static-method/basic/input.js

```diff
@@ -0,0 +1,25 @@
+class Cl {
+  static staticMethod2() {
+    return Cl.#privateStaticMethod();
+  }
+
+  static #privateStaticMethod() {
+    return 1017;
+  }
+
+  static staticMethod() {
+    return Cl.#privateStaticMethod();
+  }
+
+  static privateStaticMethod() {
+    return Cl.#privateStaticMethod();
```

Comment:
Can you also add a test for `this.#privateStaticMethod`?

Reply:
I have one here: https://github.com/babel/babel/pull/9446/files#diff-f49de9a84ff7fd3d1b84c3e6a076410f